### PR TITLE
Medical Damage - Improve description of "DamageThreshold"

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title

These settings also define damage needed to kill units by "Sum of Trauma" which is not reflected in their description:
https://github.com/acemod/ACE3/blob/3daf78441b87fb7d2995859f4e05699d19d5492d/addons/medical_damage/functions/fnc_determineIfFatal.sqf#L42-L44

### CODE REVIEW

File: addons/medical_damage/stringtable.xml [hunk 1]
```diff
@@ -15,15 +15,15 @@
...
             <French>Définit la quantité de dégâts qu'un joueur peut subir avant de perdre connaissance.</French>
             <Russian>Устанавливает количество урона, которое может получить игрок, прежде чем потеряет сознание.</Russian>
             <Portuguese>Define a quantidade de dano que um jogador pode receber antes de ficar inconsciente.</Portuguese>
             <Chinese>設定玩家在無意識前能承受多少傷害。</Chinese>
             <Italian>Imposta la quantità di danno che un giocatore può ricevere prima di perdere conoscenza.</Italian>
             <Czech>Nastavuje kolik poškození hráč může obdržet než upadne do bezvědomí.</Czech>
-            <Polish>Definiuje ilość obrażeń jaką może przyjąć gracz przed straceniem przytomności.</Polish>
+            <Polish>Definiuje ilość obrażeń jaką może przyjąć gracz przed straceniem przytomności oraz śmierci gdy "Suma urazów" jest włączona.</Polish>
             <Spanish>Fijar la cantidad de daño que un jugador puede recivir antes de caer inconsciente</Spanish>
             <German>Legt die Höhe des Schadens fest, den ein Spieler erhalten kann, bevor er ohnmächtig wird.</German>
```
Comment (bux) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <German>Legt die Höhe des Schadens fest, den ein Spieler erhalten kann, bevor er ohnmächtig wird (oder stirbt, wenn "Summe aller Traumata" aktiviert ist).</German>
```

File: addons/medical_damage/stringtable.xml [hunk 2]
```diff
@@ -41,15 +41,15 @@
...
             <French>Définit la quantité de dégâts qu'une unité IA peut subir avant de perdre connaissance.</French>
             <Russian>Устанавливает количество урона, которое может получить ИИ, прежде чем потеряет сознание.</Russian>
             <Portuguese>Define a quantidade de dano que uma IA pode receber antes de ficar inconsciente.</Portuguese>
             <Chinese>設定AI在無意識之前能承受多少傷害</Chinese>
             <Italian>Imposta la quantità di danno che un'unità AI può ricevere prima di perdere conoscenza.</Italian>
             <Czech>Nastavuje kolik poškození AI může obdržet než upadne do bezvědomí.</Czech>
-            <Polish>Definiuje ilość obrażeń jaką może przyjąć AI przed straceniem przytomności</Polish>
+            <Polish>Definiuje ilość obrażeń jaką może przyjąć AI przed straceniem przytomności oraz śmierci gdy "Suma urazów" jest włączona.</Polish>
             <Spanish>Fijar la cantidad de daño que la IA puede recivir antes de caer inconsciente</Spanish>
             <German>Legt die Höhe des Schadens fest, den eine KI-Einheit erhalten kann, bevor sie ohnmächtig wird.</German>
```
Comment (bux) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <German>Legt die Höhe des Schadens fest, den eine KI-Einheit erhalten kann, bevor sie ohnmächtig wird (oder stirbt, wenn "Summe aller Traumata" aktiviert ist).</German>
```

File: addons/medical_damage/stringtable.xml (hunk 1 above)
Comment (jonpas) [reply addons/medical_damage/stringtable.xml]: What's up with GitHub showing XML diffs like 1 space is removed?

Comment (veteran29) [reply addons/medical_damage/stringtable.xml]: I'll add the parenthesis to polish and english too. Makes the description more clear.

**Comment by veteran29** [comment]
@Elgin675 @maxpipoka can you have a look at this and suggest changes for your languages?

@BaerMitUmlaut can you ping translators for other langs?

File: addons/medical_damage/stringtable.xml [hunk 3]
```diff
@@ -15,17 +15,17 @@
             <German>Schwelle für kritischen Spielerschaden</German>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_Description">
-            <English>Sets the amount of damage a player can receive before going unconscious.</English>
+            <English>Sets the amount of damage a player can receive before going unconscious (or dying if "Sum of Trauma" is enabled).</English>
```
Comment (SilentSpike) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <English>Sets the amount of damage a player can receive before going unconscious (and dying if "Sum of Trauma" is enabled).</English>
```

File: addons/medical_damage/stringtable.xml [hunk 4]
```diff
@@ -41,17 +41,17 @@
             <German>Schwelle für kritischen KI-Schaden</German>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_AIDamageThreshold_Description">
-            <English>Sets the amount of damage an AI unit can receive before going unconscious.</English>
+            <English>Sets the amount of damage an AI unit can receive before going unconscious or dying when "Sum of Trauma" is enabled.</English>
```
Comment (SilentSpike) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <English>Sets the amount of damage an AI unit can receive before going unconscious (and dying when "Sum of Trauma" is enabled).</English>
```

File: addons/medical_damage/stringtable.xml [hunk 5]
```diff
@@ -15,17 +15,17 @@
             <German>Schwelle für kritischen Spielerschaden</German>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_Description">
-            <English>Sets the amount of damage a player can receive before going unconscious.</English>
+            <English>Sets the amount of damage a player can receive before going unconscious (or dying if "Sum of Trauma" is enabled).</English>
             <Japanese>プレイヤーが気絶になる前に受けられるダメージ量を決定できます。</Japanese>
             <French>Définit la quantité de dégâts qu'un joueur peut subir avant de perdre connaissance.</French>
```
Comment (Elgin675) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <French>Définit la quantité de dégâts qu'un joueur peut subir avant de perdre connaissance (ou mourir, si l'option "Somme des traumatismes" est sélectionnée).</French>
```

File: addons/medical_damage/stringtable.xml [hunk 6]
```diff
@@ -41,17 +41,17 @@
             <German>Schwelle für kritischen KI-Schaden</German>
         </Key>
         <Key ID="STR_ACE_Medical_Damage_AIDamageThreshold_Description">
-            <English>Sets the amount of damage an AI unit can receive before going unconscious.</English>
+            <English>Sets the amount of damage an AI unit can receive before going unconscious (or dying when "Sum of Trauma" is enabled).</English>
             <Japanese>AI が気絶になる前に受けられるダメージ量を決定できます。</Japanese>
             <French>Définit la quantité de dégâts qu'une unité IA peut subir avant de perdre connaissance.</French>
```
Comment (Elgin675) [inline addons/medical_damage/stringtable.xml]:
```suggestion
            <French>Définit la quantité de dégâts qu'une unité IA peut subir avant de perdre connaissance (ou mourir, si l'option "Somme des traumatismes" est sélectionnée).</French>
```

**Comment by veteran29** [comment]
> if we may be changing the sum of trauma behaviour in a way that changes how it interacts with this setting (as that particular lethal damage mechanism has received a lot of mixed feedback).

If it's not guaranteed that this will be changed before next minor hotfix/release it's still worth it for me as this information is critical to understand that this setting can be used to solve issue with some vests.

Eg. with some of rhs vests it's almost impossible to score critical hits to torso but this can be alleviated by using "Either" mode and setting low critical damage threshold.

**Pull request merged**